src/components/Button.jsx

```diff
@@ -74,14 +74,32 @@ function Core({
 
   if (display === 'link') {
     roleStyles = {
+      textTransform: 'none',
       background: 'none!important',
       border: 'none',
       padding: '0!important',
       color: theme.palette.text.primary,
       textDecoration: 'underline',
       cursor: 'pointer',
       backgroundColor: 'transparent',
+      padding: 0,
+      textAlign: 'right',
+      justifyContent: 'right',
+      fontSize: '14px',
     };
+    return (
+      <button
+        type="button"
+        style={{ ...roleStyles, ...style }}
+        {...rest}
+      >
+        {loading ? (
+          <CircularProgress size={24} style={{ color: 'white' }} />
```

Comment:
i know it's silly with white and black but let's use `theme.palette.common.white` here. trying to keep off-theme colors out of the codebase.

Reply:
I took the liberty of doing this everywhere.